src/Application/Renderer/Template/Action/SourcecodeHandler.php

```diff
@@ -37,7 +37,7 @@ public function __construct(Analyzer $analyzer)
      */
     public function __invoke(Action $action)
     {
-        $artifact = $action->getRenderPass()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
+        $artifact = $action->getRenderContext()->getDestination() . '/' . ltrim($action->getDestination(), '\\/');
```

Comment:
Artefact and Artifact mean the same thing and are both acceptable; generally speaking Artifact is used by the North American inhabitants, while Artefact is considered to be the British spelling. I will do a search for artifact and change it to artefact